.circleci/config.yml

```diff
@@ -43,7 +43,11 @@ jobs:
           username: user
           password: pass
           EOF
-    - run: make
+    - run:
+        command: make
+        environment:
+          # Run garbage collection more aggresively to avoid getting OOMed.
+          GOGC: "20"
```

Comment:
Am I right that CircleCI gives us an 8gb machine? https://circleci.com/docs/2.0/executor-types/

I am surprised that we are being OOMed in that case.

Reply:
No we got 4GB because we use the Docker executor with the free plan. We could use a machine executor but it takes more time to start. IMO we need to ask the CNCF for a paid account that would allow to use bigger [resources](https://circleci.com/docs/2.0/configuration-reference/#resource_class). I can bring it up during our next sync meeting.

Reply:
I'm surprised we even hit 4gb, to be honest.